articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md

```diff
@@ -0,0 +1,201 @@
+<!-- markdownlint-disable MD041 MD002 -->
+
+## Install the SDK
+
+You can install the [Auth0 Vue SDK](https://github.com/auth0/auth0-vue) using npm:
+
+```bash
+npm install @auth0/auth0-vue@beta
+```
+
+## Registering the plugin
+
+In order to use the SDK in your Vue application, you will need to register its plugin with your Vue application by passing the return value of `createAuth0` to `app.use()`.
+
+```js
+import { createAuth0 } from '@auth0/auth0-vue';
+
+const app = createApp(App);
+
+app.use(
+  createAuth0({
+    domain: '<AUTH0_DOMAIN>',
+    client_id: '<AUTH0_CLIENT_ID>',
+    redirect_uri: '<MY_CALLBACK_URL>'
+  })
+);
+
+app.mount('#app');
+```
+
+Adding the plugin will register our SDK using both `provide` and `app.config.globalProperties`, allowing the SDK to be used with both the Composition API and Options API.
+
+## Log in to the App
+
+In order to add login to your application you can use the `loginWithRedirect` function that is exposed on the return value of `useAuth0`, which you can access in your component's `setup` function.
+
+```html
+<template>
+  <div>
+    <button @click="login">Log in</button>
+  </div>
+</template>
+<script>
+  import { useAuth0 } from '@auth0/auth0-vue';
+
+  export default {
+    setup() {
+      const { loginWithRedirect } = useAuth0();
+
+      return {
+        login: () => {
+          loginWithRedirect();
+        }
+      };
+    }
+  };
+</script>
+```
+
+Calling `loginWithRedirect` will redirect the user to Auth0, and redirect them back to the `redirect_uri` (provided when calling `createAuth0()`) after entering their credentials.
+
+### Using the Options API
+In case of the Options API, you can use the same `loginWithRedirect` method from the global `$auth0` property through your component's `this`.
+
+```html
+<template>
+  <div>
+    <button @click="login">Log in</button>
+  </div>
+</template>
+
+<script>
+  export default {
+    methods: {
+      login() {
+        this.$auth0.loginWithRedirect();
+      }
+    }
+  };
+</script>
+```
+
+## Display the User's Profile
```

Comment:
Use same as React, Angular etc. quickstarts:

```suggestion
## Show User Profile Information
```

Reply:
Also, in the spirit of being consistent with other similar quickstarts, the high-level order is:

* Login
* Logout
* User Profile